driver_headers_mbox.py

```diff
@@ -23,6 +25,6 @@
     last_uid = check_validity(False, json_header_filename=unclean_headers_filename)
     print("Last valid UID in JSON file:", last_uid)
     # remove_duplicate_headers(json_header_filename=unclean_headers_filename)
-    # remove_invalid_references(input_json_filename=unclean_headers_filename, output_json_filename=headers_filename, ref_toggle=True)
-    # generate_edge_list(nodelist_filename=nodelist_filename, edgelist_filename=edgelist_filename, json_filename=unclean_headers_filename)
+    # remove_invalid_references(input_json_filename=unclean_headers_filename, output_json_filename=clean_headers_filename, ref_toggle=True)
+    # generate_edge_list(nodelist_filename=nodelist_filename, edgelist_filename=edgelist_filename, threads_json_filename=unclean_headers_filename)

```

Comment:
Why are we passing unclean headers here rather than the cleaned ones?

Reply:
Will make the changes.

Reply:
In remove_duplicate_headers, the unclean header should be passed or the clean one?

Reply:
Should be unclean because it is a part of the header data cleaning process. So the output should be clean headers for remove_invalid_references and the input unclean_headers_filename. So that is correct.

Reply:
Okay, and the same case with remove_invalid_references right?

Reply:
I'm talking about generate_edge_list